Packs/Securonix/Integrations/Securonix/Securonix.py

```diff
@@ -2513,14 +2579,20 @@ def list_lookup_table_entries(client: Client, args: Dict[str, Any]) -> Tuple:
     max_records = arg_to_number(args.get('max', '15').strip() or '15')
     offset = arg_to_number(args.get('offset', '0').strip() or '0')
     page_num = arg_to_number(args.get('page_num', '1').strip() or '1')
+    sort = args.get('sort', '').strip()
+    order = args.get('order', 'asc').strip().lower() or 'asc'
 
     # Validate required parameters.
     if not name:
         raise ValueError('Lookup table name is a required argument.')
 
+    # Validate order argument.
+    if order and order not in ['asc', 'desc']:
```

Comment:
For future PRs: It's better to check whether a value appears in a set, a it's much efficient :)
```suggestion
    if order and order not in {'asc', 'desc'}:
```